project/.travis/before_install_docs.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env sh
+set -ev
```

Comment:
I prefer keep this file even if is empty.

With this, people know that those files are managed by dev-kit and not by the project.